build/helpers/coverage.js

```diff
@@ -1,6 +1,20 @@
+import chalk from "chalk";
 import * as fs from "fs";
 import * as path from "path";
 import * as inspector from "inspector";
+import * as notify from "./notify";
+import {
+  byteLengthTotalCoverage,
+  byteLengthBlockCoverage
+} from "./coverage-heuristics/byte";
+import {
+  logicalTotalCoverage,
+  logicalBlockCoverage
+} from "./coverage-heuristics/logical";
+import {
+  arithmeticTotalCoverage,
+  arithmeticBlockCoverage
+} from "./coverage-heuristics/arithmetic";
```

Comment:
Rather than make each heuristic two separate function exports, just export a single object that describes the heuristic:

```ts
export const Arithmetic = {
  total(...)  {
    // ...
  },

  block(...) {
    // ...
  }
}
```

To use these in `coverage.js`, do:

```ts
import { Arithmetic } from "...";

const heuristics = [
  {
    ...Arithmetic,
    weight: 1 / 3
  },
  // ...
];